circle.yml

```diff
@@ -0,0 +1,9 @@
+machine:
+  node:
+    version: 5.3.0
```

Comment:
I like the idea of switching to 6.x.x or 6.0.x if we want to be more conservative.